examples/query_transactions.py

```diff
@@ -0,0 +1,234 @@
+        print('\tLeaving "{}"'.format(name))
+        return result
+
+    return tracer
+
+# Let's start defining the commands:
+@trace
+def send_transaction_and_print_status(transaction):
+    hex_hash = binascii.hexlify(IrohaCrypto.hash(transaction))
+    print('Transaction hash = {}, creator = {}'.format(
+        hex_hash, transaction.payload.reduced_payload.creator_account_id))
+    net.send_tx(transaction)
+    for status in net.tx_status_stream(transaction):
+        print(status)
+
+# For example, below we define a transaction made of 2 commands:
+# CreateDomain and CreateAsset.
+# Each of Iroha commands has its own set of parameters and there are many commands.
+# You can check out all of them here:
+# https://iroha.readthedocs.io/en/main/develop/api/commands.html
+@trace
+def create_domain_and_asset():
+    """
+    Create domain 'domain' and asset 'coin#domain' with precision 2
+    """
+    commands = [
+        iroha.command('CreateDomain', domain_id='domain', default_role='user'),
+        iroha.command('CreateAsset', asset_name='coin',
+                      domain_id='domain', precision=2)
+    ]
+# And sign the transaction using the keys from earlier:
+    tx = IrohaCrypto.sign_transaction(
+        iroha.transaction(commands), ADMIN_PRIVATE_KEY)
+    send_transaction_and_print_status(tx)
+# You can define queries 
+# (https://iroha.readthedocs.io/en/main/develop/api/queries.html) 
+# the same way.
+
+@trace
+def add_coin_to_admin():
+    """
+    Add 1000.00 units of 'coin#domain' to 'admin@test'
+    """
+    tx = iroha.transaction([
+        iroha.command('AddAssetQuantity',
+                      asset_id='coin#domain', amount='1000.00')
+    ])
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    send_transaction_and_print_status(tx)
+    # first_time, last_time = tx.reduced_payload
+    tx_tms=tx.payload.reduced_payload.created_time
+    print(tx_tms)
+    first_time, last_time = tx_tms - 1, tx_tms + 1
+    return first_time, last_time
+
+@trace
+def create_account_userone():
+    """
+    Create account 'userone@domain'
+    """
+    tx = iroha.transaction([
+        iroha.command('CreateAccount', account_name='userone', domain_id='domain',
+                      public_key=user_public_key)
+    ])
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    send_transaction_and_print_status(tx)
+
+
+@trace
+def transfer_coin_from_admin_to_userone():
+    """
+    Transfer 2.00 'coin#domain' from 'admin@test' to 'userone@domain'
+    """
+    tx = iroha.transaction([
+        iroha.command('TransferAsset', src_account_id='admin@test', dest_account_id='userone@domain',
+                      asset_id='coin#domain', description='init top up', amount='2.00')
+    ])
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    send_transaction_and_print_status(tx)
+
+
+@trace
+def userone_grants_to_admin_set_account_detail_permission():
+    """
+    Make admin@test able to set detail to userone@domain
```

Comment:
in other comments users are in quote, so make its consistent. You can surround all accounts with quote, or remove quotes. E.g.:
```suggestion
    Make `admin@test` able to set detail to `userone@domain`
```